include/seqan3/core/parallel/all.hpp

```diff
@@ -5,34 +5,16 @@
 // shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
 // -----------------------------------------------------------------------------------------------------
 
-/*!\file
- * \brief Meta-header for the parallel module.
- * \author Rene Rahn <rene.rahn AT fu-berlin.de>
- */
+ /*!\file
+  * \brief [DEPRECATED] Meta-header for the parallel module..
+  * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+  * \deprecated This header will be removed in 3.1.0. Please use seqan3/utility/parallel/all.hpp instead.
```

Comment:
```suggestion
  * \deprecated This header is deprecated and will be removed in SeqAn-3.1.0; Please \#include <seqan3/utility/parallel/all.hpp> instead.
```
(+ some linebreak)

Please use the deprecation message as described in https://github.com/seqan/product_backlog/issues/160